src/pages/aidataset/[accession_id].astro

```diff
@@ -204,8 +209,55 @@ const images = study.image_uuids.map((uuid) => exports["images"][uuid]);
     <div class="vf-section-header">
       <h3 class="vf-section-header__heading" id="annotations"> Viewable Annotations </h3>
     </div>
-    <div></div>
+    <div>
+      <table id="viewable_annotations" class="display">
+        <thead>
+          <tr>
+            <th >Preview</th>
+            <th>Filename</th>
+            <th>Download Size</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {viewable_annotation_images.map((image) =>
+            <tr>
+              <th>
+                <img class="vf-figure__image" src={image.thumbnail_uri}>
+              </th>
+              <th>{image.name}</th>
+              <th>None</th>
+              <th>
+                <a href={"/image/" + image.uuid}>View</a>
+                Download
+              </th>
+            </tr>
+          )}
+        </tbody>
+      </table>
+      </div>
     <div> </div>
   </section>
+
+
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.11.5/js/jquery.dataTables.js"></script>
```

Comment:
Haven't actually ran the code, but it looks like this is the only DataTables class that actually gets used, is this correct?

Reply:
No sadly not. It actually needs both the src source and the import to work, otherwise it doesn't build on netlify. I've tried different versions on the now deleted branch. I've taken out the source from here as it's in Baselayout anyway.